src/plugins/ucx/ucx_backend.cpp

```diff
@@ -1173,11 +1172,12 @@ nixl_status_t nixlUcxEngine::notifSendPriv(const std::string &remote_agent,
     hdr.op = NOTIF_STR;
     flags |= UCP_AM_SEND_FLAG_EAGER;
 
+    nixlSerializer sd;
     sd.addStr("name", localAgent);
     sd.addStr("msg", msg);
     // TODO: replace with mpool for performance
 
-    auto buffer = std::make_unique<std::string>(std::move(sd.exportStr()));
+    auto buffer = std::make_unique<std::string>(std::move(sd).exportStr());
```

Comment:
What does the move of nixlSeriealizer instance before exporting a string from it achieves here?